support-frontend/assets/pages/[countryGroupId]/checkout.tsx

```diff
@@ -207,7 +211,42 @@ const stripePublicKey = getStripeKey(
 const productId = query.product in productCatalog ? query.product : undefined;
 const product = productId ? productCatalog[query.product] : undefined;
 const ratePlan = product?.ratePlans[query.ratePlan];
-const price = ratePlan?.pricing[currencyKey];
+const queryPrice = query.price;
+const price = queryPrice ?? ratePlan?.pricing[currencyKey];
+const supporterPlusRatePlanPrice =
+	productCatalog.SupporterPlus.ratePlans[query.ratePlan].pricing[currencyKey];
+
+/**
+ * Is It a Contribution? URL queryPrice supplied?
+ *    If queryPrice above ratePlanPrice, in a upgrade to S+ country, invalid amount
+ */
+let isInvalidAmount = false;
+if (productId === 'Contribution' && queryPrice) {
+	const { selectedAmountsVariant } = getAmountsTestVariant(
+		countryId,
+		countryGroupId,
+		window.guardian.settings,
+	);
+	if (queryPrice < 1) {
+		isInvalidAmount = true;
+	}
```

Comment:
Just to double check this - we don't allow 50p donations?

Reply:
![image](https://github.com/guardian/support-frontend/assets/76729591/cf124446-6203-428b-8bee-189612920938)

Reply:
Think the above applies for Contributions

Reply:
Pretty sure old S+ checkout guarded against this low entry, not sure how to check tbh

Reply:
Actually if you change the Contribution amount in the paramUrl for the current checkout to £0.1 it jumps you straight to the S+ product at £10! Think marking it as invalid is fine for moment.